README.md

```diff
@@ -73,7 +74,7 @@ Plug 'SirVer/ultisnips' | Plug 'honza/vim-snippets'
 Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
 Plug 'tpope/vim-fireplace', { 'for': 'clojure' }
 
-" Using git URL
+" Any valid git URL is allowed
 Plug 'https://github.com/junegunn/vim-github-dashboard.git'
```

Comment:
How about if we move line 77 and 78 upward just after the first shorthand notation example? So it goes, "This is the shorthand notation, but you can still write a full git url".